test/orchestratord/mzcompose.py

```diff
@@ -544,52 +786,110 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         definition["materialize"] = materialize_setup[2]
 
     definition["operator"]["operator"]["tag"] = get_tag(args.tag)
+    # makes environmentd -> clusterd connections fail
+    # definition["operator"]["networkPolicies"]["enabled"] = True
+    # definition["operator"]["networkPolicies"]["internal"]["enabled"] = True
```

Comment:
Do we have a bug report for this? It should work. I typically do my testing using the cloud kind cluster and:
```yaml
networkPolicies:
  enabled: true
  internal:
    enabled: true
  ingress:
    enabled: true
  egress:
    enabled: true
```
This usually works for me.

Reply:
I have opened https://github.com/MaterializeInc/database-issues/issues/9696